Integrations/EWSv2/EWSv2.py

```diff
@@ -2017,6 +2069,8 @@ def main():
 
         if demisto.command() == 'fetch-incidents':
             raise
+        if demisto.command() == 'ews-search-mailbox' and isinstance(e, ValueError):
+            return_error("Selected invalid field, please specify valid field name.")
```

Comment:
nice, you should specify what is the invalid field (you have it in `e`)

Reply:
The error will look like: `Selected invalid field, please specify valid field name. Unknown fieldname 'subjec' on folders '[Folder(Root(<exchangelib.account.Account object at 0x7f9413752a90>, u'[self]', u'root', 16, 0, 88, None, 'AAMkADQ0NmFkODFkLWQ4MDEtNDE4Mi1hN2NkLThmZjdmNTZjNTMxNwAuAAAAAAA4kxh+ed3JTJPMPXU3wX3aAQCyyVyFtlsUQZfBJebinpkUAAAAAAEBAAA=', 'AQAAABYAAACyyVyFtlsUQZfBJebinpkUAAAfuQHa'), 'Files', 0, 0, 0, 'IPF.Files', 'AAMkADQ0NmFkODFkLWQ4MDEtNDE4Mi1hN2NkLThmZjdmNTZjNTMxNwAuAAAAAAA4kxh+ed3JTJPMPXU3wX3aAQCyyVyFtlsUQZfBJebinpkUAAAAAAFVAAA=', 'AQAAABYAAACyyVyFtlsUQZfBJebinpkUAABHP3Dt')]' in only()` with concatenating e message